### ISSUE TITLE
Half-transparent image background for forums

### ISSUE THREAD

**Issue by TheJoshue** [NamelessMC/Nameless#1739]
**Please describe the feature in as much detail as possible**
_This is maybe something to implement to (default) template._
This is a sketch, took screenshot from https://cubedcraft.com/forum/
![image](https://user-images.githubusercontent.com/56804505/86512983-c4429300-be06-11ea-926b-d81a66843428.png)
**This red lines are highlight for what I am trying to point out - certain forum/category background**

So basically let us edit this background of a forum to something transparent, for example pattern of squares/circles (that will be partially visible / 50% opacity).

If you don't really understand what I mean by this, I can elaborate or find better example for what I mean.
Literally customizing background/adding image banner as a forum/category background.

**Comment by StormyIceLeopard** [comment]
Great idea, but sadly I think that might be a template design rather than a NamelessMC feature.
Not sure if it will get implemented into the release.

**Comment by Derkades** [comment]
I'm not sure if such a feature is worth implementing as a user configurable option. If you want this you can get it by writing your own template

**Comment by StormyIceLeopard** [comment]
That's what I feared.

**Comment by samerton** [comment]
Unfortunately this is out of scope for Nameless. We provide you with a choice of basic templates, which then comes down to the community owner to develop their own theme or hire someone to do it instead.

**Comment by TheJoshue** [comment]
Well I knew it was more template-related feature than Nameless, but this can be implemented to add more options in "Settings" tab for **Default** or **DefaultRevamped** templates, nothing special as an image, just an option that can be enabled/disabled, and if enabled, it'll show additional input fields for image/whatever can be a background of forum/category.
